README.md

```diff
@@ -74,10 +74,12 @@ composer require spatie/valuestore
 
 ## Usage
 
-To create a Valuestore use the `make`-method.
+To create a Valuestore use the `make` method. You can also pass in an array as the optional second argument to store values in the Valuestore with it.
```

Comment:
Let's give the user the simplest example first and after that simplest example introduce the second parameter.